### ISSUE TITLE
azdev linter fails with ModuleNotFoundError

### ISSUE THREAD

**Issue by adewaleo** [Azure/azure-cli-dev-tools#35]
call to azdev linter fails. I don't mind taking up this issue when I make linter updates. If this isn't high priority.

```
(cli-venv) [05:23 PM] Work-Mac:azure-cli tosin$ azdev linter -r faulty_help_example_parameters_rule --ci

==============
| CLI Linter |
==============

Initializing linter with command table and help files...

 Results 
=========

No module named 'azdev.operations.linter.rules'
Traceback (most recent call last):
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/knack/invocation.py", line 188, in execute
    cmd_result = parsed_args.func(params)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/knack/commands.py", line 105, in __call__
    return self.handler(*args, **kwargs)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/knack/commands.py", line 212, in _command_handler
    result = op(client, **command_args) if client else op(**command_args)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/azdev/operations/linter/__init__.py", line 100, in run_linter
    ci=ci_mode)
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/site-packages/azdev/operations/linter/linter.py", line 191, in run
    paths = import_module('{}.rules'.format(PACAKGE_NAME)).__path__
  File "/Users/tosin/Repos/cli-venv/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'azdev.operations.linter.rules'
(cli-venv) [05:24 PM] Work-Mac:azure-cli tosin$ 

```

**Comment by tjprescott** [comment]
Thanks @adewaleo. This is fixed in PR #36.

**Comment by tjprescott** [comment]
PR #36 merged.
